covertToWordsNewVersion.js

```diff
@@ -0,0 +1,195 @@
+/*
+Create a function called convertToWords(number) it will receive a number and it will return a string  with the number written in letters.
+input: 1
+output: one
+First make it work with numbers from 0 to 10
+*/
+
+function convertToWords(number) {
+    let singleDigitsNumbersArray = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+
+    let singleDigitsNumbersNames = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"];
+
+    let TwoDigitsNumbersEnding = "teen";
+    let twentyAndPlus = "twenty";
+    let thirtyAndPlus = "thirty";
+    let fortyAndPlus = "forty";
+    let fiftyAndPlus = "fifty";
+    let sixtyAndPlus = "sixty";
+    let seventyAndPlus = "seventy";
+    let eightyAndPlus = "eighty";
+    let ninetyAndPlus = "ninety";
+    let oneHundredAndPlus = "one hundred"
+    let lastDigit = number - parseInt((number / 10)) * 10;
+
+    if (number < 10) {
```

Comment:
instead of having nested `if` blocks you can use [a switch expressions ](https://www.w3schools.com/js/js_switch.asp) for using [condition in switch ](https://stackoverflow.com/a/9055603/5533820)